components/jstraceable_derive/lib.rs

```diff
@@ -149,21 +149,15 @@ fn js_traceable_derive(s: synstructure::Structure) -> proc_macro2::TokenStream {
     let mut asserts = quote!();
     let match_body = s.each(|binding| {
         for attr in binding.ast().attrs.iter() {
-            match attr.parse_meta().unwrap() {
-                syn::Meta::Path(ref path) | syn::Meta::List(syn::MetaList { ref path, .. }) => {
-                    if path.is_ident("no_trace") {
-                        asserts.extend(assert_not_impl_traceable(&binding.ast().ty));
-                        return None;
-                    } else if path.is_ident("custom_trace") {
-                        return Some(quote!(<crate::dom::bindings::trace::CustomTraceable>::trace(#binding, tracer);));
-                    }
-                },
-                syn::Meta::NameValue(syn::MetaNameValue { ref path, .. }) => {
-                    // if reason provided we can skip JSTraceable check
-                    if path.is_ident("no_trace") {
-                        return None;
-                    }
-                },
+            if attr.path().is_ident("no_trace") {
+                // if no reason is provided to `no_trace`
+                if !matches!(attr.meta, syn::Meta::NameValue(_)) {
+                    // assert that binded type does not implement traceable
```

Comment:
```suggestion
                    // Assert that the bound type does not implement traceable.
```

Reply:
I am not sure this is called bound type. I might be wrong.

To be clear this is type of field in struct or wrapped type of enum variant.